app/frontend/components/domains/permit-application/review-permit-application-screen.tsx

```diff
@@ -64,12 +64,12 @@ export const ReviewPermitApplicationScreen = observer(() => {
   })
 
   return (
-    <Box as="main" overflow="hidden" h="full">
+    <Box as="main" overflow="hidden" h="full" id="review-permit-application">
       <Flex
         id="permitHeader"
         position="sticky"
         top={0}
-        zIndex={10}
+        zIndex={11}
```

Comment:
we should do a review of zIndex later, try to work off / extend the default theme in chakra to make things a bit more sane 💫

Reply:
yes I agree

Reply:
created https://hous-bssb.atlassian.net/browse/BPHH-976